### PR TITLE
other: Improve french translations

### PR DESCRIPTION
**Pull request by Soudaha1**
### SUMMARY
Improved french translations.

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [x] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by etr2460** [comment]
Looks like this needs a rebase, this morning I merged a format change to the translation files, so this should actually be a much smaller PR now

File: superset/translations/fr/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -17,16 +17,5616 @@ msgid ""
...
+
+#: superset-frontend/src/welcome/DashboardTable.jsx:131
+msgid "An error occurred while fetching Dashboards"
+msgstr "Une erreur s'est produite lors de la récupération des tableaux de bord"
+
+#: superset-frontend/src/welcome/Welcome.jsx:87
+#: superset-frontend/src/welcome/Welcome.jsx:92
+msgid "Recently Viewed"
+msgstr "Vu récemment"
+=======
```
Comment (etr2460) [inline superset/translations/fr/LC_MESSAGES/messages.po:5629]: This looks like a merge artifact still :/ maybe something got messed up when resolving conflicts?

Comment (DovAlduin) [reply superset/translations/fr/LC_MESSAGES/messages.po]: yes, but it should be ok now :)

**Comment by etr2460** [comment]
Cool, I think this is almost here. To make sure the translations work on the client side, I think you need to run the following:
```
npm install -g po2json
po2json -d superset -f jed1.x superset/translations/fr/LC_MESSAGES/messages.po superset/translations/fr/LC_MESSAGES/messages.json
```

Let me know if you can do that, or if you want me to push a commit onto this PR finishing up the translation process

**Comment by etr2460** [comment]
Also, perhaps @mistercrunch can review some of the translations as I believe he speaks French

**Comment by etr2460** [comment]
It looks like CI is still failing here, possibly because of the JSON formatting?

**Comment by DovAlduin** [comment]
It look likes a lint error but I do not know why

**Comment by etr2460** [comment]
The output is:
```
> prettier --check ../superset/translations/**/LC_MESSAGES/*.json

Checking formatting...
../superset/translations/fr/LC_MESSAGES/messages.json
Code style issues found in the above file(s). Forgot to run Prettier?
```

can you try running: `npm run clean-translation` from the `superset-frontend` directory? That should fix the lint errors

**Pull request closed without merging**